docker/airflow-python/delete_all_airflow_connections.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+"""
+Removing all current airflow connections
+"""
+
+from airflow import settings
+from airflow.models import Connection
+
+session = settings.Session()
```

Comment:
I'm a big fan of the create session `contextmanager`. I'm not against using the plain session here, it is just a short lived script, but I like to give good examples for people to use the `create_session()`
```suggestion
from airflow.utils.db import create_session
with create_session() as session:
session = settings.Session()
    session.query(Connection).delete()
```

Reply:
Done. In both scripts

Reply:
Awesome! 👍